web/app/dataset/create_summaries.py

```diff
@@ -0,0 +1,86 @@
+from collections import defaultdict
+
+from dataset.models import DataSummary
+from dataset.models import Vestiging, CitoScores, LeerlingNaarGewicht
+from dataset.models import SchoolAdvies, SchoolWisselaars, ToegewezenSubsidie
+
+from dataset.models import LeerlingLeraarRatio
+
+
+def create_summaries():
+    """Create a summary for all vestigingen (what data is present)."""
+    DataSummary.objects.all().delete()
+    _create_with_brin6_data()
+    _update_with_brin4_data()
+
+
+def _create_with_brin6_data():
+    summaries = defaultdict(dict)
+
+    # assumption, these models have 'brin' and 'vestigingnummer' fields
+    # and if they represent yearly measurements they have a 'jaar' field
+    MODELS = [
+        Vestiging,
+        CitoScores,
+        LeerlingNaarGewicht,
+        SchoolAdvies,
+        SchoolWisselaars,
+        ToegewezenSubsidie
+    ]
+
+    for model in MODELS:
+        is_yearly = hasattr(model, 'jaar')
+
+        queryset = model.objects.all() \
+            .filter(brin__isnull=False) \
+            .filter(vestigingsnummer__isnull=False)
+
+        objects = list(queryset)
+        for obj in objects:
+            _key = '{}{:02d}'.format(obj.brin, obj.vestigingsnummer)
+            summary = summaries[_key][model.__name__] if model.__name__ in summaries[_key] else {}
+
+            if is_yearly:
+                entry = {obj.jaar: True}
+            else:
+                entry = {'present': True}
+            summary.update(entry)
```

Comment:
Why not use the same = if else as above?

Reply:
I will do it for consistency, ternary expressions in Python are not as common as in Javascript though.